spec/actor_spec.rb

```diff
@@ -327,6 +334,19 @@
             .to raise_error(ServiceActor::ArgumentError, expected_message)
         end
       end
+
+      context "when a custom class is specified" do
+        let(:expected_message) do
+          "The \"name\" output on " \
+            "\"SetWrongTypeOfOutputWithCustomArgumentErrorClass\" must " \
+              "be of type \"String\" but was \"#{1.class.name}\""
```

Comment:
We can simplify the expected here a bit:

```suggestion
              "be of type \"String\" but was \"Integer\""
```

Reply:
Done